cypress/integration/shared.js

```diff
@@ -192,6 +198,112 @@ export const waitForClusterInstallation = () => {
   cy.contains('div.pf-c-progress__description', 'Installed', { timeout: CLUSTER_CREATION_TIMEOUT });
 };
 
+export const loginOCM = (userName, password) => {
```

Comment:
This file is becoming mess. Can we separate OCM-only things to a file i.e. 'ocm.js'?